tests/optim/core/test_loss.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
+from typing import List
+
+import numpy as np
+import torch
+
+import captum.optim._core.loss as opt_loss
+from captum.optim.models import collect_activations
+from tests.helpers.basic import BaseTest, assertArraysAlmostEqual
+from tests.helpers.basic_models import BasicModel_ConvNet_Optim
+
+CHANNEL_ACTIVATION_0_LOSS = 1.3
+CHANNEL_ACTIVATION_1_LOSS = 1.3
+
+
+def get_loss_value(
```

Comment:
Looks like this function should have a return type hint of `-> Union[int, float, np.ndarry]:`, unless there were issues with adding a return type here?